internal/series/series.go

```diff
@@ -53,6 +53,7 @@ var Series = map[string]SeriesInfo{
 	"zesty":   {true, Ubuntu, false, 0},
 	"artful":  {true, Ubuntu, true, 1.105},
 	"bionic":  {true, Ubuntu, true, 1.15},
+	"cosmic":  {true, Ubuntu, true, 1.107},
```

Comment:
do we really want to prioritise it this much already?

Reply:
No, it should have a priority of 0 and a comment saying to update the priority when it's released.